internal/cache/controller_obj_cache.go

```diff
@@ -1931,12 +1931,11 @@ func (c *AviObjCache) PopulateL4PolicySetToCache(client *clients.AviClient, clou
 
 func (c *AviObjCache) AviPopulateAllStringGroups(client *clients.AviClient, cloud string, StringGroupData *[]AviStringGroupCache, nextPage ...NextPage) (*[]AviStringGroupCache, int, error) {
 	var uri string
-	akoUser := lib.AKOUser
 
 	if len(nextPage) == 1 {
 		uri = nextPage[0].NextURI
 	} else {
-		uri = "/api/stringgroup/?" + "&include_name=true&created_by=" + akoUser
+		uri = "/api/stringgroup/?" + "&include_name=true&name.contains=" + "ako-gw-" + lib.GetClusterName()
```

Comment:
Can we use **lib.GetAKOUser()** instead of hardcoding the prefix here.

Reply:
I will add it as part of my http policy changes.

Reply:
lets there are two string groups on a controller `ako-gw-cluster` and `ako-gw-cluster1`. With this api call, we will get both of it. It should be good to restrict to only string group belongs to that cluster.